modules/apps/commerce/commerce-product-api/src/main/java/com/liferay/commerce/product/service/CPAttachmentFileEntryLocalService.java

```diff
@@ -293,6 +295,18 @@ public Folder getAttachmentsFolder(
 	public List<CPAttachmentFileEntry> getCPAttachmentFileEntries(
 		int start, int end);
 
+	@Transactional(propagation = Propagation.SUPPORTS, readOnly = true)
+	public List<CPAttachmentFileEntry> getCPAttachmentFileEntries(
+			long cpDefinitionId, Boolean galleryEnabled,
```

Comment:
why Boolean obj and not primitive?

Reply:
Because if `null`, I don't apply the filter at all.